<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>constantsManager</class>
 <widget class="QDialog" name="constantsManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="buttonsLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonAddGroup">
       <property name="text">
        <string>Add Group</string>
       </property>
       <property name="icon">
        <iconset resource="frida.qrc">
         <normaloff>:/icons/document-edit.svg</normaloff>:/icons/document-edit.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonAddValue">
       <property name="text">
        <string>Add value</string>
       </property>
       <property name="icon">
        <iconset resource="frida.qrc">
         <normaloff>:/icons/document-edit.svg</normaloff>:/icons/document-edit.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Import Constants</string>
       </property>
       <property name="icon">
        <iconset resource="frida.qrc">
         <normaloff>:/icons/document-import.svg</normaloff>:/icons/document-import.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Export Constants</string>
       </property>
       <property name="icon">
        <iconset resource="frida.qrc">
         <normaloff>:/icons/document-export.svg</normaloff>:/icons/document-export.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonDone">
       <property name="text">
        <string>Done</string>
       </property>
       <property name="icon">
        <iconset resource="frida.qrc">
         <normaloff>:/icons/document-close.svg</normaloff>:/icons/document-close.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget1">
      <layout class="QVBoxLayout" name="verticalGroups">
       <item>
        <widget class="QLabel" name="labelGroups">
         <property name="text">
          <string>Constant Groups</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableGroups">
         <property name="editTriggers">
          <set>QAbstractItemView::DoubleClicked</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>21</number>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget2">
      <layout class="QVBoxLayout" name="verticalValues">
       <item>
        <widget class="QLabel" name="labelValues">
         <property name="text">
          <string>Constant Values</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableConstantValues">
         <property name="editTriggers">
          <set>QAbstractItemView::DoubleClicked</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>21</number>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="frida.qrc"/>
 </resources>
 <connections/>
</ui>
